src/frontend/src/lib/components/ui/SnowBackground.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+	import Confetti from 'svelte-confetti';
+	import snow1 from '$lib/assets/snow-1.svg'
+	import snow2 from '$lib/assets/snow-2.svg'
+	import snow3 from '$lib/assets/snow-3.svg'
```

Comment:
in this folder you can change the assets

Reply:
Thanks I updated assets